docs/concepts/persistence.md

```diff
@@ -74,12 +74,13 @@ To ensure that your important data is retained, you may want to use a pinning se
 
 Some available pinning service providers are:
 
-- [Axel](https://www.axel.org/2019/07/23/qa-with-the-developers-of-axel-ipfs/)
-- [Eternum](https://www.eternum.io/)
+- [Web3.Storage](https://web3.storage/)
```

Comment:
Should we put this list in alphabetical order?

Reply:
We probably could. In this first run, I thought to put the ones that have full IPFS compatibility first (including DHT providing). 

As it turns out Axel is not even offering IPFS anymore.